src/stories/components/MainMenu/MainMenu.js

```diff
@@ -0,0 +1,122 @@
+import './styles.scss';
+
+import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import { ReactComponent as DashboardIcon} from
+  '../../../images/icon-dashboard.svg';
+import { ReactComponent as ExperimentsIcon} from
+  '../../../images/icon-experiments.svg';
+import { ReactComponent as AccountIcon} from
+  '../../../images/icon-account.svg';
+import { ReactComponent as LogoutIcon} from
+  '../../../images/icon-logout.svg';
+
+/**
+ * Component for main menu.
+ *
+ * @component
+ * @return {object} (
+ *   <MainMenu />
+ * )
+ */
+
+export const MainMenu = () => {
+  // List of button links/icons we currently support
+  const icons = ['Dashboard', 'Experiments', 'Account', 'Logout'];
+
+  // Hard coding the color dark teal
+  const darkTeal = '#008489';
+
+  // Returns SVG react component based on icon name
+  const getIcon = (icon) => {
+    if (icon === 'Dashboard') {
+      return (<DashboardIcon className='menu-icon' />);
+    } else if (icon === 'Experiments') {
+      return (<ExperimentsIcon className='menu-icon'/>);
+    } else if (icon === 'Account') {
+      return (<AccountIcon className='menu-icon'/>);
+    } else if (icon === 'Logout') {
+      return (<LogoutIcon className='menu-icon'/>);
+    }
+  };
```

Comment:
You should probably use a `switch` statement here.

Reply:
Will do! Thanks :D